pkg/inventory/inventory-client_test.go

```diff
@@ -82,7 +82,7 @@ func TestGetClusterInventoryInfo(t *testing.T) {
 				if err != nil {
 					t.Fatalf("unexpected error received: %s", err)
 				}
-				if !object.SetEquals(tc.localObjs, clusterObjs) {
+				if !object.IsSetEqual(tc.localObjs, clusterObjs) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.